test/mongodb.test.js

```diff
@@ -38,7 +38,7 @@ describe('lazyConnect', function() {
     });
 
     ds.on('error', function(err) {
-      err.message.should.containEql('ECONNREFUSED');
+      err.message.should.match(/connect ECONNREFUSED/);
```

Comment:
Yeah, let's not bikeshed too much here. It was just my personal preference to check for the error code because that is unlikely to change. But checking for ECONNREFUSED is fine since it makes the test more clear anyways.